README.md

```diff
@@ -1,5 +1,22 @@
 This app is a stand-alone [FHIR](http://fhir.hl7.org/) resource validator. Using this app, you can validate a FHIR resource against an optional FHIR Profile.
 
+## Running the FHIR Validator App in development
+### Requirements:
+* Ruby (the version specified in `.ruby-version`) with `bundler` installed
+* Node `12` with `npm` installed
+* Docker (to run the FHIR validator wrapper)
+
+### Setup
+* `bundle install`
+* `npm install`
```

Comment:
It might be obvious, but it couldn't hurt to add that these commands should be run from the root of the cloned repo

Reply:
Good catch! Always better to be explicit rather than make assumptions.